pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -576,6 +579,10 @@ self: super: {
     passthru.python3Dependencies = ps: with ps; [ beancount ];
   });
 
+  vim-clang-format = super.vim-clang-format.overrideAttrs(old: {
+    propagatedBuildInputs = [ clang-tools ];
+  });
```

Comment:
I would prefer patching, and avoid adding stuff to a user's path

Reply:
ok, tracking.

My experience falls a bit short here, and I didn't see any other examples in `overrides.nix` of how one would patch in a dependency like that 😅 

For simplicity's sake I've opted for *just* the standard plugin for now, and I'm betting a large majority of users for this plugin will already have `clang-format` in their environment.

If you're not happy with this approach I'm happy to figure out patching, please point to any docs/examples and I'll got work through it 😄

Reply:
Hmm, I can't find any great examples either, but I'm fine with refining it in a different PR.

I think most people will figure out that they need `clang-format` available for it to work.

maybe propagating is the solution, and I've been reviewing too many python packages